server/routes/place/html.py

```diff
@@ -262,10 +266,48 @@ def is_seo_experiment_enabled(place_dcid: str, category: str,
+# >>> DEV_PLACE_ALL_US_STATE_DCIDS = ["geoId/01","geoId/02","geoId/04","geoId/05","geoId/06","geoId/08","geoId/09","geoId/10","geoId/11","geoId/12","geoId/13","geoId/15","geoId/16","geoId/17","geoId/18","geoId/19","geoId/20","geoId/21","geoId/22","geoId/23","geoId/24","geoId/25","geoId/26","geoId/27","geoId/28","geoId/29","geoId/30","geoId/31","geoId/32","geoId/33","geoId/34","geoId/35","geoId/36","geoId/37","geoId/38","geoId/39","geoId/40","geoId/41","geoId/42","geoId/44","geoId/45","geoId/46","geoId/47","geoId/48","geoId/49","geoId/50","geoId/51","geoId/53","geoId/54","geoId/55","geoId/56","geoId/72"]
+# >>> DEV_PLACE_EXPERIMENT_COUNTRY_DCIDS = random.sample(DEV_PLACE_ALL_COUNTRY_DCIDS, int(len(DEV_PLACE_ALL_COUNTRY_DCIDS) * 0.1))
+# >>> DEV_PLACE_EXPERIMENT_US_STATE_DCIDS = random.sample(DEV_PLACE_ALL_US_STATE_DCIDS, int(len(DEV_PLACE_ALL_US_STATE_DCIDS) * 0.1))
+DEV_PLACE_EXPERIMENT_COUNTRY_DCIDS: List[str] = [
+    'country/TLS', 'country/HUN', 'country/VEN', 'country/JAM', 'country/RWA',
+    'country/GGY', 'country/NGA', 'country/COD', 'country/COG', 'country/SVN',
+    'country/LSO', 'country/LBN', 'country/LCA', 'country/NFK', 'country/TTO',
+    'country/SGP', 'country/PYF', 'country/PRK', 'country/RUS', 'country/LVA',
+    'country/SUR', 'country/PRY', 'country/IND', 'country/MDV'
+]
+DEV_PLACE_EXPERIMENT_US_STATE_DCIDS: List[str] = [
+    'geoId/56', 'geoId/04', 'geoId/41', 'geoId/20', 'geoId/37'
+]
+DEV_PLACE_EXPERIMENT_DCIDS: Set[str] = set(DEV_PLACE_EXPERIMENT_COUNTRY_DCIDS +
+                                           DEV_PLACE_EXPERIMENT_US_STATE_DCIDS)
+
+
+def is_dev_place_experiment_enabled(place_dcid: str, locale: str,
+                                    request_args: MultiDict[str, str]) -> bool:
+  """Determine if dev place experiment should be enabled for the page"""
+  # Force dev place experiment for testing
+  if request_args.get("force_dev_places") == "true":
+    return True
+  # Disable dev place experiment for testing
+  if request_args.get("disable_dev_places") == "true":
+    return False
+  # Experiment is enabled for English pages for countries and US states in the experiment group
```

Comment:
as a fail safe, can we return false if staging or prod until we do some internal testing

Reply:
Done